Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "PreallocationTools"
 uuid = "d236fae5-4411-538c-8e31-a6e3d9e00b46"
 authors = ["Chris Rackauckas <accounts@chrisrackauckas.com>"]
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
this can be a patch release.

Reply:
Oh yes, sure. Didn’t know about the convention. I’ll add a commit for this and will put a test in (the example from the issue), so that codecov is also fine.